docs/docs/introduction.md

```diff
@@ -4,10 +4,18 @@ slug: /
 
 # W&B Weave
 
-:::tip
-For a limited time, the new W&B Inference service is included in your free tier. W&B Inference provides access to leading open-source foundation models via API and the Weave [Playground](./guides/tools/playground.md). 
-- [Developer documentation](./guides/integrations/inference.md)
-- [Product page](https://wandb.ai/site/inference) 
+:::tip[🚀 New to Weave? Start with W&B Inference!]
+**No API keys needed!** Try Weave immediately with our Inference service - powerful open-source models included in your free tier.
+
+👉 **[Get Started with W&B Inference](/quickstart-inference)** - Build your first traced LLM app in minutes
```

Comment:
This whole note is a bit large for a note. Maybe keep the main CTA and then link to the QuickStart and the guide.